frontend/src/metabase/lib/query.js

```diff
@@ -517,7 +518,7 @@ var Query = {
                 display_name: field[1],
                 name: field[1],
                 // TODO: we need to do something better here because filtering depends on knowing a sensible type for the field
-                base_type: "IntegerField",
+                base_type: "type/Integer",
```

Comment:
feel like this would be way more readable if it was a TYPE.Integer instead of a bare string